corehq/util/couch.py

```diff
@@ -245,4 +245,10 @@ def _iter_update(doc_ids, try_num):
                "change or remove the '_id' field?".format(fn.__name__) +
                ", ".join(results.error_ids))
         raise IterUpdateError(results, msg)
+
+    if verbose:
+        print "couldn't find {} docs".format(len(results.not_found_ids))
+        print "ignored {} docs".format(len(results.ignored_ids))
+        print "deleted {} docs".format(len(results.deleted_ids))
+        print "updated {} docs".format(len(results.updated_ids))
```

Comment:
I like that idea.  I haven't used logging that much actually, are you picturing that I remove the verbose flag and always have these here as `logger.debug` calls, then to view this output in a management command, say, you'd import the logger and set it's level to Debug?  Is there some kind of `with log_level.DEBUG` statement?